### ISSUE TITLE
Wrong S3 bucket website_endpoint and website_domain attributes in China regions

### ISSUE THREAD

**Issue by vivanov-dp** [terraform-providers/terraform-provider-aws#9378]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

Terraform v0.12.4
provider.aws v2.19.0

### Affected Resource(s)

* aws_s3_bucket

### Terraform Configuration Files

A simple bucket:

```hcl
provider "aws" {
    region      = "cn-northwest-1"
}

resource "aws_s3_bucket" "website_build" {
    bucket = local.bucket_name
    policy = data.template_file.policy_bucket_public.rendered

    website {
        index_document = "index.html"
    }

    tags = {
        Name = "${var.prefix}-website-build"
    }
}
```

### Expected Behavior

The proper endpoint is `http://<bucket>.s3-website.cn-northwest-1.amazonaws.com.cn`

### Actual Behavior

```
$ terraform.exe console
?[J> aws_s3_bucket.website_build.website_endpoint
<bucket>.s3-website.cn-northwest-1.amazonaws.com

?[J> aws_s3_bucket.website_build.website_domain
s3-website.cn-northwest-1.amazonaws.com

?[J> aws_s3_bucket.website_build.bucket_regional_domain_name
<bucket>.s3.cn-northwest-1.amazonaws.com.cn
```

Terraform is missing the .cn at the end of the domain for the website_endpoint and website_domain attributes. For comparison - bucket_regional_domain_name has a proper value.

### Steps to Reproduce

1. `terraform apply`

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.21.0 of the Terraform AWS Provider, later this week. 👍 Thanks to @antonio-osorio for the implementation.

**Comment by bflad** [comment]
This has been released in [version 2.21.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2210-july-25-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

For further feature requests, documentation updates, or bug reports with this functionality, please create a [new GitHub issue](https://github.com/terraform-providers/terraform-provider-aws/issues/new/choose) following the template for triage. Thanks!
